### ISSUE TITLE
Standalone `flake8-trio` could read config files like `flake8`

### ISSUE THREAD

**Issue by Zac-HD** [Zac-HD/flake8-trio#189]
This is a nice convenience over specifying them on the command-line every time.  Could be implemented as an `--config=` argument pointing to an ini file for easy implementation; any more than that and we should probably just hook flake8's config handling if we don't mind the runtime dep when standalone.

**Comment by jakkdl** [comment]
yeah I had it listed in #124, although given that most of that task list is the low prio "convert X to libcst and make it autofix" the general functionality ones probably deserve their own issues.
If running it through pre-commit you do sort of have config support atm, but I'll look at various diff ways of configuring :+1:
